source/mainnet/smart-contracts/best-practices/index.rst

```diff
@@ -0,0 +1,12 @@
+. _sc-best-practices:
+
+==============
+Best practices
+==============
+
+Here, you can find best practices for smart contract development, audit, information about common pitfalls and security vulnerabilities, and how to avoid them.
```

Comment:
I think we should remove this page for now. Then we can add it once we have more than one page of best practices. I don't see much benefit in having it now, and it just adds an extra click for the user.

Reply:
I wanted to create a structure that could be filled-in with new content, and possibly write some short guidelines on how to contribute. So I could drive the process (and contribute myself), but also encourage contributions by others.
We can discuss how to organise the process on one of the stand-ups.

Reply:
Ok, let's keep the structure as is then.